### PR TITLE
Reorg extdata

### PR DESCRIPTION
**Pull request by markomi**
I have reorganized the extdata directory. Thanks for reviewing.

This resolves issue #65 .

### CODE REVIEW

File: tests/testthat/test-read_export_options_languages.R [hunk 1]
```diff
@@ -2,68 +2,80 @@ context("export options internationalization")
...
 # GERMAN
 # german long file names
-export_path_long_de <- system.file("extdata", "s_export_CSV-xls_CTU05_longnames_sep_ref_german.zip",
+export_path_long_de <- system.file("extdata",  "sT_exports", "longnames",
+                                   "s_export_CSV-xls_CTU05_long_ref_miss_de_utf8.zip",
                                   package = "secuTrialR")
 sT_export_long_de <- read_secuTrial_raw(data_dir = export_path_long_de)
 # german short file
-export_path_short_de <- system.file("extdata", "examples_short", "s_export_CSV-xls_CTU05_20190710-110208_de.zip",
+export_path_short_de <- system.file("extdata",  "sT_exports", "shortnames",
```
Comment (PatrickRWright) [inline tests/testthat/test-read_export_options_languages.R]: double space

File: tests/testthat/test-read_export_options_languages.R [hunk 2]
```diff
@@ -2,68 +2,80 @@ context("export options internationalization")
...
+                                    "s_export_CSV-xls_CTU05_short_meta_ref_miss_de_utf8.zip",
                                     package = "secuTrialR")
 sT_export_short_de <- read_secuTrial_raw(data_dir = export_path_short_de)
 
 # ITALIAN
 # italian long file name
-export_path_long_it <- system.file("extdata", "s_export_CSV-xls_CTU05_longnames_sep_ref_italian.zip",
+export_path_long_it <- system.file("extdata",  "sT_exports", "longnames",
```
Comment (PatrickRWright) [inline tests/testthat/test-read_export_options_languages.R]: double space

File: tests/testthat/test-read_export_options_languages.R [hunk 3]
```diff
@@ -2,68 +2,80 @@ context("export options internationalization")
...
+                                   "s_export_CSV-xls_CTU05_long_ref_miss_it_utf8.zip",
                                   package = "secuTrialR")
 sT_export_long_it <- read_secuTrial_raw(data_dir = export_path_long_it)
 # italian short file name
-export_path_short_it <- system.file("extdata", "examples_short", "s_export_CSV-xls_CTU05_20190710-105847_it.zip",
+export_path_short_it <- system.file("extdata",  "sT_exports", "shortnames",
```
Comment (PatrickRWright) [inline tests/testthat/test-read_export_options_languages.R]: double space

File: tests/testthat/test-read_export_options_languages.R [hunk 4]
```diff
@@ -2,68 +2,80 @@ context("export options internationalization")
...
-export_path_short_fr <- system.file("extdata", "examples_short", "s_export_CSV-xls_CTU05_20190710-105945_fr.zip",
+export_path_short_fr <- system.file("extdata", "sT_exports", "shortnames",
+                                    "s_export_CSV-xls_CTU05_short_meta_ref_miss_fr_utf8.zip",
                                     package = "secuTrialR")
 sT_export_short_fr <- read_secuTrial_raw(data_dir = export_path_short_fr)
 
 # SPANISH
 # spanish long file name
-export_path_long_es <- system.file("extdata", "s_export_CSV-xls_CTU05_longnames_sep_ref_spanish.zip",
+export_path_long_es <- system.file("extdata",  "sT_exports", "longnames",
```
Comment (PatrickRWright) [inline tests/testthat/test-read_export_options_languages.R]: double space

File: tests/testthat/test-read_export_options_languages.R [hunk 5]
```diff
@@ -2,68 +2,80 @@ context("export options internationalization")
...
+                                   "s_export_CSV-xls_CTU05_long_ref_miss_es_utf8.zip",
                                   package = "secuTrialR")
 sT_export_long_es <- read_secuTrial_raw(data_dir = export_path_long_es)
 # spanish short file name
-export_path_short_es <- system.file("extdata", "examples_short", "s_export_CSV-xls_CTU05_20190710-105648_es.zip",
+export_path_short_es <- system.file("extdata",  "sT_exports", "shortnames",
```
Comment (PatrickRWright) [inline tests/testthat/test-read_export_options_languages.R]: I will stop marking them now ;)

Comment (markomi) [reply tests/testthat/test-read_export_options_languages.R]: Well spotted :) thanks!

**Comment by PatrickRWright** [comment]
Thank you @markomi

**Pull request merged**